opencontrol.yaml

```diff
@@ -4,8 +4,12 @@ metadata:
   description: "Certifications in OpenControl format."
   maintainers:
     - Shawn Wells (shawn@redhat.com)
+    - Mark Shoger (mshoger@redhat.com)
 certifications:
   - ./fedramp-low.yaml
   - ./fedramp-moderate.yaml
   - ./fedramp-high.yaml
   - ./dhs-4300a.yaml
+  - ./ICD503-low.yaml
+  - ./ICD503-moderate.yaml
+  - ./ICD503-high.yaml
```

Comment:
- Because ICD 503 has a space in the name, should these files be ``icd-503-*``?
- Filenames should be lowercase